expo-maps/ios/GoogleMapsView.swift

```diff
@@ -21,4 +21,19 @@ public final class GoogleMapsView: UIView {
   required init?(coder: NSCoder) {
     fatalError("init(coder:) has not been implemented")
   }
+  
+  func mapType(mapTypeName: String) -> Void {
+    var mapViewType: GMSMapViewType
+    switch mapTypeName {
+    case "hybrid":
+      mapViewType = .hybrid
+    case "satelite":
+      mapViewType = .satellite
+    case "terain":
+      mapViewType = .terrain
+    default:
+      mapViewType = .normal
+    }
+    self.mapView.mapType = mapViewType
+  }
```

Comment:
We can do it better ;) 
Our API allows you to automatically cast from string to enum. You can read more about this feature here: https://github.com/expo/expo/blob/7511bd66bebdbc8edff7817b687fe0e2d86e202b/docs/pages/modules/module-api.md#enums